src/bundle/Resources/public/scss/_alloyeditor-anchor-edit.scss

```diff
@@ -0,0 +1,50 @@
+.ez-ae-anchor-edit {
+    display: flex;
+    padding: calculateRem(8px) 0 calculateRem(8px) calculateRem(8px);
+
+    &__input-wrapper {
```

Comment:
What about moving it inside the `&__input {` selector? In fact, it looks like it should be a separate component, but it's acceptable use case here.

Reply:
Are you suggesting using this structure:
```scss
 &__input {
    /*...*/
    &-wrapper {/*...*/}
    &-label {/*...*/}
}
```
? I have bad experience with splitting selectors in this way. I have once searched for a selector and it took me a while, because _element_ name was split in halve. 
For me the most natural and easy to comprehend approach is to split class name in 3: _block_, _element_, _modifier_, like it was done here. 🙂